packages/@ourworldindata/grapher/src/stackedCharts/StackedBarChart.tsx

```diff
@@ -321,6 +339,25 @@ export class StackedBarChart
             )
         )
 
+        const hoverPoints = series.map((series) => {
+            const point = series.points.find(
+                (bar) => bar.position === hoverTime
+            )
+            return {
+                seriesName: series.seriesName,
+                seriesColor: series.color,
+                point,
+            }
+        })
+        const [positivePoints, negativePoints] = partition(
+            hoverPoints,
+            ({ point }) => (point?.value ?? 0) >= 0
+        )
```

Comment:
Future idea: Would actually be quite cool if we could do a three-way partition into [positive, zero, negative] points here